src/libImaging/Convert.c

```diff
@@ -631,6 +690,24 @@ i2rgb(UINT8* out, const UINT8* in_, int xsize)
     }
 }
 
+static void
+i2hsv(UINT8* out, const UINT8* in_, int xsize)
+{
+    int x;
+    INT32* in = (INT32*) in_;
+    for (x = 0; x < xsize; x++, in++, out+=4) {
+        out[0] = 0;
+        out[1] = 0;
+        if (*in <= 0)
+            out[2] = 0;
+        else if (*in >= 255)
+            out[2] = 255;
+        else
+            out[2] = (UINT8) *in;
```

Comment:
Let's bracket these single-liners for safety.

Reply:
Okay, done